client/stylesheets/_toCat.scss

```diff
@@ -5219,7 +5233,7 @@ On click the images will be displayed at normal size to complete the effect
 
 .profileSettingsButton {
     width: 70%;
-    margin: 10px 0 100px 0 !important;
+    margin: 25px auto 100px auto !important
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Declaration should be terminated by a semicolon